soh/soh/Enhancements/randomizer/randomizer_entrance_tracker.cpp

```diff
@@ -687,6 +687,12 @@ void DrawEntranceTracker(bool& open) {
                 UIWidgets::Tooltip("Reveal the \"To\" entrance for undiscovered entrances");
                 UIWidgets::PaddedEnhancementCheckbox("Show \"From\"", "gEntranceTrackerShowFrom", true, false);
                 UIWidgets::Tooltip("Reveal the \"From\" entrance for undiscovered entrances");
+                UIWidgets::Spacer(2.0f);
+                bool disableHideReverseEntrances = OTRGlobals::Instance->gRandomizer->GetRandoSettingValue(RSK_DECOUPLED_ENTRANCES) == RO_GENERIC_ON;
+                static const char* disableHideReverseEntrancesText = "This option is disabled because \"Decouple Entrances\" is enabled.";
+                UIWidgets::EnhancementCheckbox("Hide reverse", "gEntranceTrackerHideReverseEntrances",
+                                              disableHideReverseEntrances, disableHideReverseEntrancesText);                
+                UIWidgets::Tooltip("Hide reverse entrance transitions when Decouple Entrances is off");
```

Comment:
If you use `UIWidgets::PaddedEnhancementCheckbox` you don't need the spacer, and it excepts the same params, plus extra params for adding a top or bottom space.

The last param allows you to specify the default value. Check the method signature to see what I mean.

I think this checkbox also should go on the left column since that contains the settings related to the list.

Reply:
Yeah, that makes sense. Done.